### ISSUE TITLE
Manual ore mining drops ore blocks intead of GT ore chunks

### ISSUE THREAD

**Issue by quadrohedron** [TerraFirmaGreg-New-Horizons/TFG-Modpack-1.12.x#198]
### Version modpack Used

Default

### Environment

Singleplayer

### New Worlds

Yes

### Expected Behavior

Breaking an ore block with a pickaxe or a mining hammer (supposedly) should drop GregTech ore chunks.

The assumption of the "right" drops is based on the fact that horse grindstone ore processing works only with those GT chunks, as well as the fact that it was the actual behaviour in previous versions of the modpack.

### Actual Behavior

Using those tools on an ore block drops the blocks themselves instead.

![image](https://github.com/TerraFirmaGreg-New-Horizons/TFG-Modpack-1.12.x/assets/8057737/e98a5f11-f257-4bdd-bd22-167855391012)

### Steps to Reproduce

1) Break an ore block with a pick or a mining hammer.
2) Observe the block drop.

### Additional Information

This bug of the 4.3.0 release seems to be well known on the Discord server, but I decided to make this report anyways, seeing that it's not formally logged on the issue tracker.

**Comment by Wicpar** [comment]
I have the same in the latest extended version

**Comment by SpeeeDCraft** [comment]
fixed in dev, thanks for opening this issue

**Comment by SpeeeDCraft** [comment]
No, only when new ver of tfg:nh will be released

**Comment by barti170** [comment]
> No, only when new ver of tfg:nh will be released

any idea when this fix will come out ? Coz if soon, I might just come back to the modpack later

**Comment by zortordor** [comment]
> No, only when new ver of tfg:nh will be released

Any info on when it comes out? I thought about messing with customnpcs or an autoconverter, but it ain't gonna be worth it if the new version drops soon.

**Comment by SpeeeDCraft** [comment]
we couldn't fix it in 4.3.1 unfortunately, for some reason the fix doesn't work and we are still working and studying this error, sorry

**Comment by zortordor** [comment]
> we couldn't fix it in 4.3.1 unfortunately, for some reason the fix doesn't work and we are still working and studying this error, sorry

Automatically converting the blocks to chunks on pickup with the Auto Oredict Converter could work as a temoprary fix

https://www.curseforge.com/minecraft/mc-mods/auto-ore-dictionary-converter

**Comment by barti170** [comment]
or for now until you fix it. Add a recipe to Horse Grindstone that accepts the ore blocks. Outcome would be the same, players would still get "2 impure pile of xxx dusts" (unless ore blocks used to drop more chunks from ore then maybe 2-4 dust). This was quest progression would be feel good. 
Not gonna lie, I crafted that horse grindstone hoping to get a lilttle more out of these blocks just to find out they only accept chunks, which cannot drop right now.

**Comment by Teteros** [comment]
> or for now until you fix it. Add a recipe to Horse Grindstone that accepts the ore blocks. Outcome would be the same, players would still get "2 impure pile of xxx dusts" (unless ore blocks used to drop more chunks from ore then maybe 2-4 dust). This way quest progression would be feel right.

Good idea, I've got it to work, here are the lines you can edit in file `scripts\GregTech\AutomaticRecipes\OreChunks\GeneratingRecipes.zs`
to add quern/grindstore recipes for all gregtech ores alongside the chunks:

```diff
--- GeneratingRecipes.zs        2023-08-07 17:48:52.612350300 +0100
+++ GeneratingRecipes_.zs       2023-08-07 17:45:38.568218195 +0100
@@ -11,17 +11,34 @@
 
 import mods.dropt.Dropt;
 
-val oreChunk as OrePrefix = OrePrefix.getPrefix("oreChunk");
-
-oreChunk.generateRecipes(function(orePrefix as OrePrefix, material as Material) {
-
-    // Дробилка TFC
-    Quern.addRecipe("tfg/gregtech/ore_chunk/" + material.name, Utils.item(orePrefix, material), Utils.item("crushed", material) * 1);
-
-    // Конская дробилка
-    Grindstone.add(Utils.item(orePrefix, material), Utils.item("dustImpure", material) * 2, 10, false);
-
-} as IOreRecipeHandler);
+function processOres(prefix as string){
+    val op as OrePrefix = OrePrefix.getPrefix(prefix);
+       val prefixName = prefix == "oreChunk" ? "ore_chunk" : prefix;
+       op.generateRecipes(function(orePrefix as OrePrefix, material as Material) {
+
+               // Дробилка TFC
+               Quern.addRecipe("tfg/gregtech/" + prefixName + "/" + material.name, Utils.item(orePrefix, material), Utils.item("crushed", material) * 1);
+
+               // Конская дробилка
+               Grindstone.add(Utils.item(orePrefix, material), Utils.item("dustImpure", material) * 2, 10, false);
+
+       } as IOreRecipeHandler);
+}
+
+processOres("ore");
+processOres("oreChunk");
+
+//val oreChunk as OrePrefix = OrePrefix.getPrefix("oreChunk");
+
+//oreChunk.generateRecipes(function(orePrefix as OrePrefix, material as Material) {
+//
+//    // Дробилка TFC
+//    Quern.addRecipe("tfg/gregtech/ore_chunk/" + material.name, Utils.item(orePrefix, material), Utils.item("crushed", material) * 1);
+//
+//    // Конская дробилка
+//    Grindstone.add(Utils.item(orePrefix, material), Utils.item("dustImpure", material) * 2, 10, false);
+//
+//} as IOreRecipeHandler);
 
 // Fix strange bug with oreQuartzite
 // Centrifuge
```
Here's the edited [GeneratingRecipes.zs](https://cdn.discordapp.com/attachments/812676262908395530/1138151085006139493/GeneratingRecipes.zs) for pack version 4.3.1 so you can drop it in your instance if that's easier.

**Comment by Xikaro** [comment]
fixed in dev
